src/analyzer.ts

```diff
@@ -18,16 +20,14 @@ export class Analyzer extends AnalyzerGen {
 
 		this.analyzerProcess.stdout.on("data", (data: Buffer) => {
 			let message = data.toString();
-			console.log(`RCV: ${message}`);
+			if (verbose && message.trim().length != 0) console.log(`<== ${message}`);
```

Comment:
I'm really not a fan of `if`s all on one line like this, easy to miss things (possibly Kiln's bad indenting in code reviews is to blame, it sometimes indents completely differently to the source file!).

I don't go as far as putting braces around single-line ifs, though I've discovered in Dart I've had to start doing that to stop the formatter doing this!

Maybe I'll learn to deal with it; I don't know :D